components/http_foundation.rst

```diff
@@ -377,6 +377,14 @@ method takes an instance of
 You can clear a cookie via the
 :method:`Symfony\\Component\\HttpFoundation\\ResponseHeaderBag::clearCookie` method.
 
+You can also create a 
+:class:`Symfony\\Component\\HttpFoundation\\Cookie` object from a raw header value using :method:`Symfony\Component\HttpFoundation\Cookie::fromString`.
+
+.. tip::
```

Comment:
You're right, I'd say it's also a way to use symfony API, I guess it does not hurt to keep it, any other opinion on the matter?